### PR TITLE
Update IM (copy+clone load fix)

### PR DESCRIPTION
**Pull request by joker806**
https://keboola.slack.com/archives/C02C3GZUS/p1636960939001400

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
         "keboola/billing-api-php-client": "^0.1",
         "keboola/configuration-variables-resolver": "^1.0.0",
         "keboola/dockerbundle": "dev-odin-library-3@dev",
-        "keboola/input-mapping": "13.2.0",
+        "keboola/input-mapping": "13.2.2",
```
Comment (pivnicek) [inline composer.json:26]: should be able to release the hard version?  like ^13.2.x?

Comment (joker806) [reply composer.json:26]: OFC! wrong version pushed :facepalm:

Comment (joker806) [reply composer.json]: fixed, thanks

**Review by pivnicek — APPROVED** [review]
LGTM

**Pull request merged**